ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/CVEsTable.tsx

```diff
@@ -92,6 +93,9 @@ type CVEsTableProps = {
     getSortParams: UseURLSortResult['getSortParams'];
     isFiltered: boolean;
     filteredSeverities?: VulnerabilitySeverityLabel[];
+    showExceptionMenuItems: boolean;
+    selectedCves: ReturnType<typeof useSet<string>>;
+    cveTableActionHandler: (opts: { type: CveExceptionRequestType; cves: string[] }) => void;
```

Comment:
Is this same as `ExceptionRequestModalOptions` type from CVEsTableContainer.tsx file?

Reply:
Yes good catch, I'll move that down into the child component and import it here.